config/routes.rb

```diff
@@ -390,6 +390,8 @@
 
       get '/by_mail/enter_code' => 'by_mail/enter_code#index', as: :verify_by_mail_enter_code
       post '/by_mail/enter_code' => 'by_mail/enter_code#create'
+      get '/by_mail/enter_code/try_again_later' => 'by_mail/enter_code_rate_limited#index',
```

Comment:
Is there a reason not to make the route /rate_limited for consistency?
```suggestion
      get '/by_mail/enter_code/rate_limited' => 'by_mail/enter_code_rate_limited#index',
```

Reply:
I was thinking if the route was user facing do we want to tell them that they are rate limited?

Reply:
Oh, good point. I'm not sure what we want to do about that. Maybe ask our PM/PO?